docs/what-is-celo/using-celo/bridged_tokens/update_token_list.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+import urllib.request
+import json
+
+tokenlist_url = "https://raw.githubusercontent.com/ethereum-optimism/ethereum-optimism.github.io/refs/heads/master/optimism.tokenlist.json"
+tokenlist = json.load(urllib.request.urlopen(tokenlist_url))
+
+output = []
+for token in tokenlist["tokens"]:
+    if token["chainId"] != 42220:
+        continue
+
+    output.append(
+        [
+            token["name"],
+            token["symbol"],
+            f"[{token['address']}](https://celoscan.io/token/{token['address']})",
+        ]
+    )
+
+with open("tokens.md", "w") as f:
+    f.write("| Symbol | Token Name | L2 Contract Address |\n")
```

Comment:
Can you include the L1 address in the table?

Reply:
Luckily, the `opTokenId` can be used as a unique identifier across chains. L1 addresses are included now. I added both addresses into a single cell because the table got too wide if I put the in a separate column.